teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+    @specialization = specialization
+    super(age, name, parent_permission: parent_permission)
+  end
+
+  def can_use_services?
+    'true'
```

Comment:
[OPTIONAL]
- [ ] Also here instead of returning the string `"true"` It would be great to return instead the boolean variable `true`